webpack.config.js

```diff
@@ -1,9 +1,7 @@
-/* eslint-env node */
-
 const webpack = require('webpack');
 
 module.exports = {
-  entry: ['whatwg-fetch', 'promise-polyfill/src/polyfill', './src/index.js'],
```

Comment:
Why using replacing `'whatwg-fetch', 'promise-polyfill/src/polyfill'` by `babel-regenerator-runtime` ?

Reply:
`babel-regenerator-runtime` compiles async/await. `whatwg-fetch` and `promise-polyfill/src/polyfill` are no longer needed because of `babel-preset-env`. We can test on IE to make sure.

Reply:
@arnaudlewis babel-preset-env is now working with polyfill.io (usebuiltIns adds like 5000 lines of code). polyfill.io runs a small script on browsers that don't support fetch/promise/etc